src/org/usfirst/frc/team2335/robot/subsystems/DriveTrain.java

```diff
@@ -2,39 +2,74 @@
 
 import org.usfirst.frc.team2335.robot.Robot;
 
-import edu.wpi.first.wpilibj.RobotDrive;
 import edu.wpi.first.wpilibj.TalonSRX;
+import edu.wpi.first.wpilibj.Victor;
 import edu.wpi.first.wpilibj.command.Subsystem;
 
 public class DriveTrain extends Subsystem
 {
-	RobotDrive drive;
+	Victor leftDrive, rightDrive;
 	TalonSRX strafe;
 	
 	public DriveTrain()
 	{		
+		leftDrive = new Victor(Robot.LEFT_PORT);
```

Comment:
What exactly is this change? I see we split up the motors, was that based on the types of motors or something with the code?